configure

```diff
@@ -1567,6 +1597,16 @@ case "${ARCH}" in
                 fi
             fi
 
+            check_mtune_cascadelake_compiler_flag
+
+            if test ${MTUNE_CASCADELAKE_AVAILABLE} -eq 1; then
+                avx512tune="-mtune=cascadelake"
+            else
+                if test ${MTUNE_SKYLAKE_AVX512_AVAILABLE} -eq 1; then
+                    avx512tune="-mtune=skylake-avx512"
```

Comment:
After having another look at this, I am wondering; Won't `AVX512TUNE` be missing from the Makefiles and thus never actually get used for the configure flow?

I think the best solution here is not to add `AVX512TUNE` but to rather append the chosen mtune parameter to the existing `avx512flag` variable. Those two would always be used together anyways so there is probably no real need to separate them.
That also matches better with how it is done in cmake and how configure applied the mtune before, making it a minimal change.


While you are at it, I'd appreciate it if you would rebase the whole PR on top of latest develop so the last CI tests hopefully gets fixed.

Reply:
Thanks for review! Both things are done.